packages/pigeon/lib/kotlin_generator.dart

```diff
@@ -369,6 +410,75 @@ class KotlinGenerator extends StructuredGenerator<KotlinOptions> {
     indent.newln();
   }
 
+  void _writeCodecOverflowUtilities(
+    KotlinOptions generatorOptions,
+    Root root,
+    Indent indent,
+    List<EnumeratedType> types, {
+    required String dartPackageName,
+  }) {
+    final NamedType overflowInt = NamedType(
+        name: 'type',
+        type: const TypeDeclaration(baseName: 'Int', isNullable: false));
+    final NamedType overflowObject = NamedType(
+        name: 'wrapped',
+        type: const TypeDeclaration(baseName: 'Object', isNullable: true));
+    final List<NamedType> overflowFields = <NamedType>[
+      overflowInt,
+      overflowObject,
+    ];
+    final Class overflowClass = Class(
+        name:
+            '${generatorOptions.fileSpecificClassNameComponent}$_overflowClassName',
+        fields: overflowFields);
+
+    indent.writeln('@Suppress("ClassName")');
+    _writeDataClassSignature(indent, overflowClass);
```

Comment:
As with Swift, can we add `private`?

Reply:
I added the logic, but forgot to use it...